upa/contracts/IUpaVerifier.sol

```diff
@@ -113,6 +113,12 @@ interface IUpaVerifier {
         bytes32 submissionId
     ) external view returns (uint256);
 
+    /// Get the current off-chain submission proof ids.
+    function getCurrentOffChainSubmissionProofIds()
```

Comment:
Totally get why you added this.  However this interface is intended to be the interface seen by the "user" (we should have made this clearer in the doc comment).  So, for example, the `verifyAggregatedProof` call intentionally does not appear here as it is only intended for the aggreagtor(s). The function you are adding is in the same category.

(Side note - we don't really need an interface for the aggregator, since it's reasonable to assume the aggregator is familiar with the contract details and can even cope with method changes if the contract is updated.)

Reply:
Will remove it from the interface 👍

Reply:
Done